storage/innobase/handler/handler0alter.cc

```diff
@@ -662,9 +665,6 @@ static MY_ATTRIBUTE((warn_unused_result)) bool innobase_need_rebuild(
       ha_alter_info->create_info->explicit_encryption,
       ha_alter_info->create_info->encrypt_type.str)};
 
-  const bool is_file_per_table{
-      !tablespace_is_shared_space(ha_alter_info->create_info)};
```

Comment:
ha_alter_info is created from old table(src) or the new table(dest)?
Also am bit skeptic about m_prebuilt->table. Sometimes it can be nullptr :thinking:

Reply:
Also there is ctx->prebuilt..

Reply:
ha_alter_info->create_info is created based on old table and then can be modify by alter code. ha_alter_info consist of modification to the old table.

Reply:
ok thanks